shared/package.json

```diff
@@ -196,13 +194,17 @@
     "redux-logger": "3.0.6",
     "source-map-support": "0.5.3",
     "style-loader": "0.20.3",
-    "uglifyjs-webpack-plugin": "1.1.4",
-    "webpack": "3.10.0",
+    "uglifyjs-webpack-plugin": "1.2.4",
+    "webpack": "4.4.1",
+    "webpack-cli": "^2.0.13",
     "webpack-dashboard": "1.1.1",
-    "webpack-dev-server": "2.11.1",
+    "webpack-dev-server": "3.1.1",
     "webpack-merge": "4.1.2"
   },
   "optionalDependencies": {
     "regedit": "2.2.7"
-  }
+  },
+  "sideEffects": [
+    "*.css"
```

Comment:
style loader would get optimized out if we didn't mark it as being a side effect import